__tests__/app.test.js

```diff
@@ -9,6 +9,26 @@ beforeEach(() => {
 });
 afterAll(() => db.end());
 
+describe("api", () => {
+  test("GET:200 returns an object of available endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const endPoints = response.body.endPoints;
+        for (const endpoint in endPoints) {
+          expect(typeof endPoints[endpoint].description).toBe("string");
+          expect(Array.isArray(endPoints[endpoint].queries)).toBe(true);
+          expect(
+            typeof endPoints[endpoint].exampleResponse &&
+              endPoints[endpoint].exampleResponse !== null &&
+              !Array.isArray(endPoints[endpoint].exampleResponse)
+          ).toBe(true);
```

Comment:
This is thorough testing but it could be a bit over engineered, it might be simpler to `const / require` in the _endpoints.json_ file and assert that your response looks exactly the same as the contents of the file using `toEqual`